scripts/build_twilio_library.py

```diff
@@ -50,6 +50,18 @@ def generate(spec_folder: str, spec_files: List[str], output_path: str, language
             if language in generateForLanguages.get(spec_file):
                 generate_domain_for_language(spec_file, config_path, spec_folder, output_path, language, parent_dir)
         else: generate_domain_for_language(spec_file, config_path, spec_folder, output_path, language, parent_dir)
+    if spec_files[0] in generateForLanguages and language in generateForLanguages.get(spec_files[0]):
+        print(f'Generating {output_path} from {spec_folder}')
+        run_openapi_generator(parent_dir, language)
+        print(f'Code generation completed at {output_path}')
+    elif spec_files[0] not in generateForLanguages:
+        print(f'Generating {output_path} from {spec_folder}')
+        run_openapi_generator(parent_dir, language)
+        print(f'Code generation completed at {output_path}')
+    if language in CLEANUP_IMPORT_LANGUAGES:
+        remove_unused_imports(output_path, language)
+    if language in REMOVE_DUPLICATE_IMPORT_LANGUAGES:
+        remove_duplicate_imports(output_path, language)
 
```

Comment:
try to create a different PR which has less commits. This PR has 9 commits.
Changes looks good, I have tested code is getting generated within 20 seconds.

Reply:
It takes 20 seconds because we are passing directory, If file is passed it takes more time.

Reply:
Get approval from @AsabuHere as well.